ajax/libs/swiftclick/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "swiftclick",
+  "description": "Eliminates the 300ms click event delay on touch devices that support orientation change",
+  "author": {
+    "name": "Ivan Hayes",
+    "url": "http://www.ivanhayes.com"
+  },
+  "homepage": "https://github.com/munkychop/swiftclick",
```

Comment:
@huhgawz We will use `url` in repository attribute first, so let's remove `homepage` field point to github repo. Thanks.